web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1709,7 +1709,6 @@ public void executing(ChangeSet changeSet, int numChangeSetsToRun) {
 						}
 						catch (InputRequiredException inputRequiredEx) {
 							// TODO display a page looping over the required input and ask the user for each.
-							// 		When done and the user and put in their say, call DatabaseUpdater.update(Map);
```

Comment:
@Nuwantha leave. you just remove one sentence in the middle of a block. that doesnt make sense. either remove it all or leave it.